schema/me/consignments/submission.js

```diff
@@ -0,0 +1,100 @@
+import { GraphQLObjectType, GraphQLString, GraphQLNonNull, GraphQLBoolean } from "graphql"
+
+export const SubmissionType = new GraphQLObjectType({
+  name: "Submission",
+  description: "A work to be consigned to the user",
+  fields: {
+    id: {
+      description: "Convection id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    additional_info: {
+      description: "The type of participant, e.g. Partner or User",
+      type: GraphQLString,
+    },
+    artist_id: {
+      decription: "The gravity ID for an Artist",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    authenticity_certificate: {
+      description: "Does the artwork come with an certificate of authenticty?",
+      type: GraphQLBoolean,
+    },
+    category: {
```

Comment:
This should probably be an ENUM type since there are only certain accepted values.